woocommerce/woocommerce
# fix e2e tag paging, bump fallback versions

### Changes proposed in this Pull Request:

This PR fixes an error that occurs when all of the first page of the tags returned from the `hub.docker.com` api have been updated in the last 12 hours. The current logic assumes that there will be at least one release tag found on the first page.

As a follow up to #30529, this PR also bumps the fallback PHP & MariaDB versions.

### How to test the changes in this Pull Request:

1. Verify CI run pulled 5.7.2
2. The WP 5.8.0 tag was last updated at `2021-08-19 04:15 UTC`. 
3. If the PR is reviewed after `2021-08-19 16:15 UTC` then `npx wc-e2e docker:up` should use WP 5.8.0.
4. Re-run CI

### Changelog entry

> N/A
